.eslintrc.yaml

```diff
@@ -125,3 +127,35 @@ rules:
 
   # eslint-plugin-jsx-expressions – https://github.com/hpersson/eslint-plugin-jsx-expressions/
   jsx-expressions/strict-logical-expressions: warn
+
+  # eslint-plugin-import https://github.com/import-js/eslint-plugin-import/
+  import/first: error
+  import/no-duplicates: error
+
+  # eslint-plugin-simple-import-sort https://github.com/lydell/eslint-plugin-simple-import-sort/
+  simple-import-sort/exports: error
```

Comment:
Import alone doesn't handle this? I thought it did. I'll have to look at what we use again…

Reply:
It should, however I think the configuration was a lot more involved for module and typescript resolving?